connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -216,7 +216,7 @@ public boolean commitOffsets() {
             this.committableOffsets = CommittableOffsets.EMPTY;
         }
 
-        if (committableOffsets.isEmpty()) {
+        if (offsetsToCommit.isEmpty()) {
```

Comment:
Don't we need to update the logging in the `else` branch to use `offsetsToCommit` instead of `committableOffsets` as well?

Reply:
Yeah this is embarrassing :(  

Fixed

Reply:
Haha, no worries! Considering I wrote this buggy logic in the first place I'm at least equally embarrassed here 😅